sources/packages/forms/src/form-definitions/institutionlocation.json

```diff
@@ -1211,6 +1211,507 @@
                     "tree": false,
                     "id": "ehusb3l",
                     "dataGridLabel": false
+                },
+                {
+                    "legend": "Primary Contact",
+                    "tooltip": "",
+                    "customClass": "",
+                    "tabindex": "",
+                    "hidden": false,
+                    "disabled": false,
+                    "modalEdit": false,
+                    "key": "fieldset",
+                    "tags": [],
+                    "properties": {},
+                    "conditional": {
+                        "show": null,
+                        "when": null,
+                        "eq": "",
+                        "json": ""
+                    },
+                    "customConditional": "",
+                    "logic": [],
+                    "attributes": {},
+                    "overlay": {
+                        "style": "",
+                        "page": "",
+                        "left": "",
+                        "top": "",
+                        "width": "",
+                        "height": ""
+                    },
+                    "type": "fieldset",
```

Comment:
I believe that we stopped using the fieldsets on form.IO to create headers.
Probably this form needs to receive more styles in the future so I will leave it to you change it now or later 😉